packages/manager/src/features/CloudPulse/Overview/GlobalFilters.tsx

```diff
@@ -1,20 +1,26 @@
-import { Dashboard } from '@linode/api-v4';
+import { Grid } from '@mui/material';
 import { IconButton, Tooltip } from '@mui/material';
 import { styled } from '@mui/material/styles';
-import Grid from '@mui/material/Unstable_Grid2';
 import * as React from 'react';
 
 import Reload from 'src/assets/icons/reload.svg';
-import { WithStartAndEnd } from 'src/features/Longview/request.types';
+import { Divider } from 'src/components/Divider';
 
+import { CloudPulseDashboardFilterBuilder } from '../shared/CloudPulseDashboardFilterBuilder';
 import { CloudPulseDashboardSelect } from '../shared/CloudPulseDashboardSelect';
-import { CloudPulseRegionSelect } from '../shared/CloudPulseRegionSelect';
-import { CloudPulseResources } from '../shared/CloudPulseResourcesSelect';
-import { CloudPulseResourcesSelect } from '../shared/CloudPulseResourcesSelect';
 import { CloudPulseTimeRangeSelect } from '../shared/CloudPulseTimeRangeSelect';
 
+import type { FilterValueType } from '../Dashboard/CloudPulseDashboardLanding';
+import type { Dashboard, TimeDuration } from '@linode/api-v4';
+import type { WithStartAndEnd } from 'src/features/Longview/request.types';
+
 export interface GlobalFilterProperties {
-  handleAnyFilterChange(filters: FiltersObject): undefined | void;
+  handleAnyFilterChange(
+    filterKey: string,
+    filterValue: FilterValueType
+  ): undefined | void;
```

Comment:
```suggestion
  ):  void;
```
undefined is not required as it is not going to return anything

Reply:
done